staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -155,6 +164,9 @@ func (c completedConfig) NewWithDelegate(delegationTarget genericapiserver.Deleg
 		handledGroups:    sets.String{},
 		lister:           informerFactory.Apiregistration().InternalVersion().APIServices().Lister(),
 		APIRegistrationInformers: informerFactory,
+		enableAggregatorRouting:  c.EnableAggregatorRouting,
```

Comment:
https://github.com/kubernetes/kubernetes/pull/44742/files#r117992791 bubbles to here.  Interface for determining host.